src/BulletCollision/NarrowPhaseCollision/btGjkEpa2.cpp

```diff
@@ -41,21 +41,38 @@ namespace gjkepa2_impl
 
 	/* GJK	*/ 
 #define GJK_MAX_ITERATIONS	128
-#define GJK_ACCURARY		((btScalar)0.0001)
-#define GJK_MIN_DISTANCE	((btScalar)0.0001)
-#define GJK_DUPLICATED_EPS	((btScalar)0.0001)
+
+#ifdef BT_USE_DOUBLE_PRECISION
+	#define GJK_ACCURARY		((btScalar)1e-12)
```

Comment:
Good point Sherm. There are other typos in the code base, but it is good to address them.